pkg/model/estimator/local/benchmark_test.go

```diff
@@ -76,22 +77,25 @@ func benchmarkNtesting(b *testing.B, continerNumber int) {
 	}
 	nodeMetrics.AddNodeResUsageFromContainerResUsage(containersMetrics)
 	b.ResetTimer()
-	local.UpdateContainerEnergyByRatioPowerModel(containersMetrics, nodeMetrics)
+	var wg sync.WaitGroup
```

Comment:
no need wait group in test case?

Reply:
Just to follow the same logic of the main code.